country-a-service/epps-service/src/main/java/dk/sundhedsdatastyrelsen/ncpeh/service/DispensationAllowed.java

```diff
@@ -17,6 +17,7 @@ private DispensationAllowed() {
 
     /// Check whether dispensation is allowed. Returns an error message if it is not, otherwise null.
     ///
+    /// @param prescription the prescription to dispense. Fetching the orders for the prescription is required.
```

Comment:
```suggestion
    /// @param prescription the prescription to dispense. Must include effectuations and orders.
```
otherwise it's confusing - "fetching the orders for the prescription " could sound like something you do *with* the prescription object.